client/src/lib/slate-pointers/PointerImportNode.tsx

```diff
@@ -94,6 +106,7 @@ export class PointerImportNode extends React.Component<any, any> {
     if (!isOpen) {
       return (
         <ClosedPointerImport
+          onClick={e => this.handleClickingClosedPointer(e, this.props.nodeAsJson.data.internalReferenceId)}
```

Comment:
Could make sense to factor out to `const pointerId = this.props.nodeAsJson.data.internalReferenceId` since it's used here and below.